dspace/modules/xmlui-mirage2/src/main/webapp/themes/Mirage2/styles/_style.scss

```diff
@@ -25,4 +25,10 @@
 #aspect_submission_StepTransformer_list_submit-project .ds-add-button {
     margin-top: 10px;
     min-width: 30%;
+}
+
+.upload-rioxx-version-warning {
```

Comment:
Add a "    margin-bottom: 10px;" to the h3 element in this div so that the lines are spaced more evenly. Do the same for Mirage 1.